wlauto/workloads/skype/__init__.py

```diff
@@ -0,0 +1,81 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import time
+from wlauto import AndroidUxPerfWorkload, Parameter
+
+
+class Skype(AndroidUxPerfWorkload):
+
+    name = 'skype'
+    description = '''
+    A workload to perform standard productivity tasks within Skype.  The workload logs in to the
```

Comment:
similarly to the old `skypevideo` workload this should have instructions covering the fact that a skype acocunt needs to be setup. For voice you can just use the skype echo service but for video a second skype account that auto accepts calls is required. 

In light of this I think the workload should default to voice not video too and default the contact_name to `Echo / Sound Test Service`